src/config/constants/trading-competition/easter.ts

```diff
@@ -0,0 +1,186 @@
+export enum Tiers {
+  GOLD = 'Gold',
+  SILVER = 'Silver',
+  BRONZE = 'Bronze',
+  PURPLE = 'Purple',
+  TEAL = 'Teal',
+}
+
+export interface Achievement {
+  champion?: number
+  teamPlayer?: number
+  trophy: number
+}
+
+export interface Rank {
+  rank: string
+  tier: Tiers
+  cakePrizeInUsd: number
+  achievements: Achievement
+  hasNft: boolean
+}
+
+interface Config {
+  [key: string]: Rank[]
+}
+
+const easterPrizes: Config = {
+  1: [
+    {
+      rank: '1',
+      tier: Tiers.GOLD,
+      cakePrizeInUsd: 21000,
+      achievements: {
+        champion: 1250,
+        teamPlayer: 750,
+        trophy: 500,
+      },
+      hasNft: true,
+    },
```

Comment:
Again, this is the first round but I wanted to keep the config as loose as possible while keeping some flexibility.